packages/application/style/menus.css

```diff
@@ -32,7 +32,7 @@
 
 .lm-MenuBar-item {
   /* stylelint-disable-next-line csstree/validator */
-  padding: 0 min(calc(1.2vw - 2px), 8px);
+  padding: 0 min(calc(1.17vw - 2px), 8px); /* this value for the padding is to allow for the help dropdown to be show at 400% zoom */
```

Comment:
Great! Could you also explain the calc() expression as well. Why is it `1.17vw - 2px`?